src/languageServerInstaller.ts

```diff
@@ -9,16 +9,16 @@ import { Release, getRelease } from '@hashicorp/js-releases';
 export class LanguageServerInstaller {
 	public async install(directory: string): Promise<void> {
 		const { version: extensionVersion } = require('../package.json');
-		const lsVersionCmd = `${directory}/terraform-ls --version`;
 		const userAgent = `Terraform-VSCode/${extensionVersion} VSCode/${vscode.version}`;
-		let isInstalled = true;
+		let isInstalled = false;
 		try {
-			var { stdout, stderr } = await exec(lsVersionCmd);
-			var installedVersion = stdout || stderr;
+			var installedVersion = await getLsVersion(directory);
+			console.log(`Found installed LS ${installedVersion}`);
```

Comment:
The console log and error functions don't get seen by the user, so we should either throw the error to make sure it's handled, or use some other kind of output.

Reply:
I think in this particular case it was mostly intentional to keep it silent as from UI perspective finding LS version is an implementation detail, but I thought it might be helpful for debugging later?